docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+---
+slug: quickstart
+append_help_link: true
+title: Quickstart
+hide_title: false
+description: Learn how to set up Semgrep and scan your first repository.
+tags:
+  - quickstart
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Learn how to set up Semgrep, scan your first project for security issues, and view your findings.
+
+:::info prerequisites
+You must have Python 3.8 or later installed on the machine where the Semgrep CI is running.
+:::
+
+1. Navigate to [Semgrep](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to grant Semgrep the needed permissions and proceed.
```

Comment:
```suggestion
1. Navigate to [Semgrep Cloud Platform (SCP)](https://semgrep.dev/login), and sign up by clicking on **Sign in with GitHub** or **Sign in with GitLab**. Follow the on-screen prompts to grant Semgrep the needed permissions and proceed.
```

Alternatively: If the wordiness of Cloud Platform bothers you, we can also say "Sign into your Semgrep account".